src/main/resources/db/changelog/db.changelog-diff-20180619-1433.yaml

```diff
@@ -0,0 +1,155 @@
+databaseChangeLog:
```

Comment:
I don't think this file is included by mistake. This would be trying to do a clean install of all sequences and tables